image_op/reconstruct_traces.m

```diff
@@ -171,7 +183,11 @@ function reconstruct_traces(movie_source, ica_dir, varargin)
     pix_active = find(rec_filter>0);
     movie_portion = M(pix_active,:)';
     movie_portion(movie_portion<threshMov) = 0;
-    traces(:,cell_idx) = movie_portion * rec_filter(pix_active);  
+    trace_this = (movie_portion * rec_filter(pix_active))';  
+    % Fix baseline
```

Comment:
These operations for baseline removal are not obvious, can you explain in comments what it is doing here?